File: Sources 4.5.2/Gizmox.WebGUI.Forms.CompanionKit/Controls/ListView/Features/LabelEditPage.cs
using System;
using System.Text;
using System.Data;
using System.Drawing;
using System.ComponentModel;
using System.Collections.Generic;

using Gizmox.WebGUI.Forms;
using Gizmox.WebGUI.Common;

namespace CompanionKit.Controls.ListView.Features
{
    public partial class LabelEditPage : UserControl
    {
        public LabelEditPage()
        {
            InitializeComponent();
        }

        /// <summary>
        /// Handles the CheckedChanged event of the mobjEditCheck control.
        /// </summary>
        /// <param name="sender">The source of the event.</param>
        /// <param name="e">The <see cref="EventArgs"/> instance containing the event data.</param>
        private void mobjEditCheck_CheckedChanged(object sender, EventArgs e)
        {
            //Change ListView.LabelEdit property
            mobjListView.LabelEdit = mobjEditCheck.Checked;
        }

        /// <summary>
        /// Handles the CheckedChanged event of the RB control.
        /// </summary>
        /// <param name="sender">The source of the event.</param>
        /// <param name="e">The <see cref="EventArgs"/> instance containing the event data.</param>
        private void RB_CheckedChanged(object sender, EventArgs e)
        {
            //Set ListView.View property to selected view
            if (mobjDetailsRB.Checked)
                mobjListView.View = View.Details;
            else if (mobjListRB.Checked)
                mobjListView.View = View.List;
            else if (mobjSmallIconRB.Checked)
                mobjListView.View = View.SmallIcon;
            else
                mobjListView.View = View.LargeIcon;
        }

    }
}

File: Sources 4.5.2/Gizmox.WebGUI.Forms.CompanionKit/Controls/ListView/Features/LabelEditPage.vb
Namespace CompanionKit.Controls.ListView.Features

    ''' <summary>
    ''' 
    ''' </summary>
    Public Class LabelEditPage

		Public Sub New()

			' This call is required by the Visual WebGui UserControl Designer.
			InitializeComponent()

		End Sub
        ''' <summary>
        ''' Handles the CheckedChanged event of the mobjEditCheck control.
        ''' </summary>
        ''' <param name="sender">The source of the event.</param>
        ''' <param name="e">The <see cref="System.EventArgs"/> instance containing the event data.</param>
        Private Sub mobjEditCheck_CheckedChanged(sender As System.Object, e As System.EventArgs) Handles mobjEditCheck.CheckedChanged
            'Change ListView.LabelEdit property
            mobjListView.LabelEdit = mobjEditCheck.Checked
        End Sub

        ''' <summary>
        ''' Handles the CheckedChanged event of the RB control.
        ''' </summary>
        ''' <param name="sender">The source of the event.</param>
        ''' <param name="e">The <see cref="System.EventArgs"/> instance containing the event data.</param>
        Private Sub RB_CheckedChanged(sender As System.Object, e As System.EventArgs) Handles mobjDetailsRB.CheckedChanged, mobjListRB.CheckedChanged, mobjLargeIconRB.CheckedChanged, mobjSmallIconRB.CheckedChanged
            'Set ListView.View property to selected view
            If mobjDetailsRB.Checked Then
                mobjListView.View = View.Details
            ElseIf mobjListRB.Checked Then
                mobjListView.View = View.List
            ElseIf mobjSmallIconRB.Checked Then
                mobjListView.View = View.SmallIcon
            Else
                mobjListView.View = View.LargeIcon
            End If
        End Sub
    End Class

End Namespace